apps/innbyggers-flate/nais/nais-dev.yaml

```diff
@@ -56,7 +56,7 @@ spec:
           },
           "proxies": [
             {
-              "fromPath": "/amt-deltaker-bff",
+              "fromPath": "(/pr-\d+)?/amt-deltaker-bff",
```

Comment:
🎉

Reply:
Må denne kanskje bli oppdatert av poao-frontend workflowene for at pr-bygget skal fungere